io.github.lainsce.Notejot.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "io.github.lainsce.Notejot",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Notejot",
+    "rename-icon" : "io.github.lainsce.Notejot",
```

Comment:
Why this one is needed?

Reply:
Oh oops. Going to remove that. Was using an older manifest and renaming stuff around, missed that.